http.go

```diff
@@ -215,7 +215,12 @@ func httpGetReputation(w http.ResponseWriter, r *http.Request) {
 	// If the request is for an IP type object, consult the exception list. Currently
 	// exceptions only apply to IP objects.
 	if typestr == "ip" {
-		if isException(valstr) {
+		exc, err := isException(valstr)
+		if err != nil {
+			w.WriteHeader(http.StatusBadRequest)
```

Comment:
I think a 400 is ok, but given that we are pre validating IPs before the isException() func in here (httpGetReputation()), perhaps it should be a 500? I dont feel particularly strongly about this -- just a thought.